config/cesm/machines/config_machines.xml

```diff
@@ -510,23 +510,23 @@ This allows using a different mpirun command to launch unit tests
       </modules>
```

Comment:
You missed one.

Reply:
What did I miss? I 'm not seeing it.

Reply:
You need to scroll up a little, it doesn't show up in the PR
https://github.com/ESMCI/cime/pull/3652/files/0744804d823d5e810b24b80db36d16fb17482e13#diff-5df7fe8863138858d69aaea393fad13dR509

Reply:
I think I fixed it. Its the reference to b17 rather than b23.